lib/sdr_client/deposit/files/direct_upload_request.rb

```diff
@@ -10,19 +10,36 @@ def self.from_file(path, file_name:, content_type:)
           checksum = Digest::MD5.file(path).base64digest
           new(checksum: checksum,
               byte_size: ::File.size(path),
-              content_type: content_type || 'application/octet-stream',
+              content_type: clean_content_type(content_type),
```

Comment:
perhaps don't call it here, or fix the method to only change it if you think it's json?

Reply:
i believe it is needed in both spots, as different methods in the client call it in different places, but I didn't fully trace everything all the way through

it shouldn't have any negative impact though, as any regular content types are unaffected